src/Test/NodeTestCase.php

```diff
@@ -53,7 +48,7 @@ protected function getCompiler(?Environment $environment = null)
 
     protected function getEnvironment()
     {
-        return $this->currentEnv = new Environment(new ArrayLoader());
```

Comment:
maybe it was meant to be `??=` instead, to reuse the same instance when called multiple times in a test

Reply:
It was used when we had more methods reusing the env, but it's not that useful anymore. Maybe we can indeed keep the "cache".